markdown/developers/guidelines/requirements/identity_branding.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Identity and branding
+subtitle: Primary development must on the nf-core organisation.
+menu:
+  main:
+    weight: 5
+---
+
+The nf-core community was founded to allow different groups to collaborate on pipelines.
+At its inception, the first step was to remove instututional branding from existing workflows and migrate them to the nf-core organisation and nf-core branding.
+This was done to remove institutional and organisational ownership of pipelines and code, to encourage collaboration and openness.
+In this spirit, we require nf-core pipelines to be _owned_ by nf-core, and nf-core alone.
+
+Whilst pipelines can be forked on GitHub for **personal** development work, the nf-core repository should be the primary source for all development.
+
+Specifically:
+
+### The nf-core repository should be set as the _head_ repo
+
+Your personal / organisation repos should show on GitHub as being forked from nf-core, not the other way around.
+This is so that it's clear where the primary development location is.
+It also means that any pull-requests that are created will automatically select the nf-core repository as the target.
```

Comment:
```suggestion
It also means that any pull-requests that are created will automatically select the nf-core repository as the target, and issues are placed in a centralised place for more visibility.
```

Or something like this (as we also discussed this explictly at the team-meet)

Reply:
Being the head fork has no impact on issue creation, that I'm aware of.

I already mention in the section below to disable issues on forks.